_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -532,11 +533,11 @@ public static function is_site_verified_and_token( $request ) {
 		$uuc_settings                         = get_option( 'uuc_settings', array() );
 		$csp4                                 = get_option( 'seed_csp4_settings_content', array() );
 		if (
-			'true' === $mm_coming_soon
-			|| 1 == $under_construction_activation_status // WPCS: loose comparison ok.
-			|| ( isset( $ucp_options['status'] ) && 1 == $ucp_options['status'] ) // WPCS: loose comparison ok.
-			|| ( isset( $uuc_settings['enable'] ) && 1 == $uuc_settings['enable'] ) // WPCS: loose comparison ok.
-			|| ( isset( $csp4['status'] ) && ( 1 == $csp4['status'] || 2 == $csp4['status'] ) ) // WPCS: loose comparison ok.
+			( ( is_plugin_active( 'bluehost' ) || is_plugin_active( 'mojo-marketplace' ) || is_plugin_active( 'mojo-marketplace-wp-plugin' ) ) && 'true' === $mm_coming_soon )
```

Comment:
Instead of using `is_plugin_active`, I would recommend using `Jetpack::is_plugin_active`; it improves on the default core function by handling Multisite as well.